python/pyarrow/_parquet.pyx

```diff
@@ -1421,13 +1421,6 @@ cdef class ParquetReader(_Weakrefable):
                          .ReadColumn(column_index, &out))
         return pyarrow_wrap_chunked_array(out)
 
-    def read_schema_field(self, int field_index):
```

Comment:
@jorisvandenbossche Do you think it's fine to remove this (undocumented) method? The ParquetReader class doesn't seem to be documented as a public API...

Reply:
@jorisvandenbossche Ping.

Reply:
Sorry, missed this ping. Yes, I think it is fine to just remove this. The ParquetReader is indeed not meant as public API, although it is unfortunate (I see now ..) that we actually import this in the pyarrow.parquet namespace, instead of calling it as `_parquet.ParquetReader` (that probably something else we could fix). As a user you should use `ParquetFile`, the `ParquetReader` is quite inconvenient to construct.